snap/local/mux_select_nav_vel.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/bash
+$SNAP/opt/ros/noetic/bin/rosservice call /mux/select "/nav_vel"
```

Comment:
Couldn't we have a single `mux_select` script with an argument instead of three?

Reply:
not really. the `command-chain` entry only accept ASCII alphanumeric characters. So we cannot feed a script along with an argument